packages/kbn-optimizer/src/common/bundle.ts

```diff
@@ -23,7 +23,7 @@ import { BundleCache } from './bundle_cache';
 import { UnknownVals } from './ts_helpers';
 import { includes, ascending, entriesToObject } from './array_helpers';
 
-const VALID_BUNDLE_TYPES = ['plugin' as const];
+const VALID_BUNDLE_TYPES = ['plugin' as const, 'entry' as const];
```

Comment:
I could imagine the login and logout pages to be "entry" type bundles. The inclusion of the core in those bundles has caused a number of issues over the years and it seems there needs not be a whole "platform" loaded to render those two pages.

Reply:
OTOH, maybe it'd be better if the navigation after login was a SPA-style nav to the next app, which would be instant. Either way, I'm sort of indifferent here. I just used `entry` to make it more clear how this bundle could be used in a browser.

Reply:
I'm all about making login instant, it's really logout that has caused the most issues and it's also one of those pages that should really be as instant as possible.

Reply:
> The inclusion of plugins in those bundles has caused a number of issues over the years and it seems there needs not be a whole "platform" loaded to render those two pages.

Not sure it's feasible. That would add a limitation that security plugin cannot use other plugins API (licensing, for example) on `login` / `logout` pages. Also, it's already working with bootstrapping all the KP plugins.

Reply:
> Also, it's already working with bootstrapping all the KP plugins.

There have been a handful of nasty race conditions caused by bootstrapping all the plugins on the logout page, which have lead to security vulnerabilities, and they currently serve no purpose. So, I'm on board with "if it ain't broke, don't fix it", but I think this has broken enough times that we should look for solutions. Especially since the bugs have historically been "I clicked logout, and it said I was logged out, but when I refresh the page I was logged in"